.github/workflows/parsing-tests.yml

```
@@ -206,6 +221,91 @@ jobs:
           path: |
             **/plot_*.svg
 
+  push-logs:
+    name: Generate AST diff
+    runs-on: [self-hosted, Linux, X64]
+    container: ubuntu:jammy
+    needs:
+      - tests-read-systemverilog
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+
+      - name: Download test results
+        uses: /actions/download-artifact@v3
+        with:
+          name: parsing-logs
+          path: logs
+
+      - name: Install dependencies
+        run: |
+          apt-get update -qq
+          apt install -y git
+
+      - name: Generate push branch name
+        run: |
+          branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}
+          PUSH_BRANCH=$branch
+          if [[ ! -z "$PLUGINS_BRANCH_SPEC" ]]; then
+            PUSH_BRANCH="$PUSH_BRANCH+plugins+$PLUGINS_BRANCH_SPEC"
```

```suggestion
            PUSH_BRANCH="${PUSH_BRANCH}+{plugins}+${PLUGINS_BRANCH_SPEC}"
```
readability, especially due to `${VARNAME+}` being a thing.

---

Should ``plugins`` also have ``{}`` around it? Or you wanted to change branch name too?

---

Oh, sorry! No, it should be `${PUSH_BRANCH}+plugins+${PLUGINS_BRANCH_SPEC}`.